rpcclient/local.go

```diff
@@ -65,8 +73,82 @@ func (l *Local) ABCIQueryWithOptions(ctx context.Context, path string, data tmby
 // BroadcastTxCommit returns with the responses from CheckTx and DeliverTx.
 // More: https://docs.tendermint.com/master/rpc/#/Tx/broadcast_tx_commit
 func (l *Local) BroadcastTxCommit(ctx context.Context, tx types.Tx) (*ctypes.ResultBroadcastTxCommit, error) {
-	// needs mempool
-	panic("not implemented!")
+	subscriber := "" //ctx.RemoteAddr()
```

Comment:
What does this line mean/do?

Reply:
Added comment to clarify this.

Reply:
Interesting, I didn't know this (or maybe I forgot). Thanks for adding the comment. It clarifies that line.